### PR TITLE
^: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/^.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Caret
+
+> Bash builtin to quick substitute a string in the previous command and run the result.
+> Equivalent to `!!:s^string1^string2`.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#Event-Designators>.
+
+- Substitute `string1` with `string2` in the previous command:
```
Comment (vitorhcl) [inline pages/common/^.md:7]:
```suggestion
- Run the previous command replacing `string1` with `string2`:
```

File: pages/common/^.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+`^{{string1}}^{{string2}}`
+
+- Remove `string1` from the previous command:
+
+`^{{string1}}^`
+
+- Substitute `string1` with `string2` in the previous command and add `string3` to the end:
```
Comment (vitorhcl) [inline pages/common/^.md:15]:
```suggestion
- Replace `string1` with `string2` in the previous command and add `string3` to its end:
```

**Review by vitorhcl — COMMENTED** [review]
Thank you for your contribution! I have some suggestions

File: pages/common/^.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Caret
```
[the remark this answers is not in the document]
Comment (Managor) [reply pages/common/^.md:1]: Results is a CI error https://github.com/Managor/tldr/actions/runs/8020762879/job/21911264093

For the same reason `!.md` has "Exclamation mark" https://github.com/tldr-pages/tldr/blob/main/pages/common/!.md

**Review by fazlearefin — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**